projects/optic/src/commands/oas/captures/streams/sources/proxy.ts

```diff
@@ -196,7 +196,8 @@ export class ProxyInteractions {
     });
 
     function onAbort(e) {
-      capturingProxy.reset();
+      capturingProxy.stop();
+      transparentProxy.stop();
```

Comment:
This was being called but because the `interactions` stream wasn't being consumed, the process never exited